Mathlib/Data/Matrix/ColumnRowPartitioned.lean

```diff
@@ -34,14 +34,14 @@ def fromRows (A₁ : Matrix m₁ n R) (A₂ : Matrix m₂ n R) : Matrix (m₁ 
 
 /-- Concatenate together two matrices B₁[m × n₁] and B₂[m × n₂] with the same rows (M) to get a
 bigger matrix indexed by [m × (n₁ ⊕ n₂)] -/
-def fromColumns (B₁ : Matrix m n₁ R) (B₂ : Matrix m n₂ R) : Matrix m (n₁ ⊕ n₂) R :=
+def fromCols (B₁ : Matrix m n₁ R) (B₂ : Matrix m n₂ R) : Matrix m (n₁ ⊕ n₂) R :=
```

Comment:
These definitions should all come with `deprecated` `alias`es that preserve the old names.

I think it's probably ok not to deprecate the lemmas, but you should list all the renames in the "Moves" section of the PR.

Reply:
I have a slight preference for deprecating all the lemmas, too, but won't insist.

Reply:
All right, I'll add `deprecated` everywhere.